### PR TITLE
https://issues.redhat.com/browse/ACM-11779

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

**Review by mprahl — APPROVED** [review]

**Comment by dockerymick** [comment]
@mprahl thanks for your review and approval! I will reach out to receive a peer review from my team.

File: governance/config_policy_ctrl.adoc [hunk 1]
```diff
@@ -123,21 +123,27 @@ By default, the minimum time between evaluations for configuration policies is a
 
 | `spec.object-templates[].complianceType`
 | Required
-| Used to define the desired state of the Kubernetes object on the managed clusters. You must use one of the following verbs as the parameter value:
+a| Used to define the desired state of the Kubernetes object on the managed clusters. You must use one of the following verbs as the parameter value:
```
Comment (jc-berger) [inline governance/config_policy_ctrl.adoc]: optional, increase user focus: 

```suggestion
a| Use it to define the desired state of the Kubernetes object on your managed clusters. Use one of the following verbs as the parameter value:
```

File: governance/config_policy_ctrl.adoc [hunk 2]
```diff
@@ -123,21 +123,27 @@ By default, the minimum time between evaluations for configuration policies is a
...
+- `mustnothave`: Indicates that an object with the same fields as specified in the `objectDefinition` cannot exist.
 
 | `spec.object-templates[].metadataComplianceType`
 | Optional
 | Overrides `spec.object-templates[].complianceType` when comparing the manifest's metadata section to objects on the cluster ("musthave", "mustonlyhave"). Default is unset to not override `complianceType` for metadata.
 
 | `spec.object-templates[].recordDiff`
 | Optional
-| Specifies if and where to log the difference between the object on the cluster and the `objectDefinition` in the policy. Set to `Log` to log the difference in the controller logs or `None` to not log the difference. By default, this parameter is empty to not log the difference.
+a| Specifies if and where to display the difference between the object on the cluster and the `objectDefinition` in the policy. The following options are supported:
```
Comment (jc-berger) [inline governance/config_policy_ctrl.adoc]: optional, again with the user focus: 

```suggestion
a| Use it to specify if and where to display the difference between the object on the cluster and the `objectDefinition` in the policy. Choose from the following supported options:
```

**Review by jc-berger — APPROVED** [review]
/lgtm

File: governance/config_policy_ctrl.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/config_policy_ctrl.adoc]:
```suggestion
a| Use this parameter to define the desired state of the Kubernetes object on your managed clusters. Use one of the following verbs as the parameter value:
```

File: governance/config_policy_ctrl.adoc (hunk 2 above)
Comment (dockerymick) [reply governance/config_policy_ctrl.adoc]:
```suggestion
a| Use this parameter to specify if and where to display the difference between the object on the cluster and the `objectDefinition` in the policy. The following options are supported:
```

**Comment by dockerymick** [comment]
Thanks @jc-berger ! I've accepted most if your suggestions, merging this PR

**Pull request merged**